### PR TITLE
docs: fix hotreload tabs

### PR DESCRIPTION
**Pull request by rajamatt**
GitHub Issue (If applicable): https://github.com/unoplatform/uno-private/issues/501

## PR Type

What kind of change does this PR introduce?

- Documentation content changes (bugfix)

## What is the current behavior?

Caused by #17435, indicator section is only visible in the WinAppSDK tab:

image

## What is the new behavior?

The HotReload Indicator section is not part of the tabs anymore, it is part of the root page.

image

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by rajamatt** [comment]
Screenshot of the problem is from https://unodocsprstaging.z13.web.core.windows.net/pr-17435/index.html

**Review by agneszitte — APPROVED** [review]

**Comment by agneszitte** [comment]
@Mergifyio backport release/stable/5.3

**Review by dr1rrb — APPROVED** [review]

**Comment by agneszitte** [comment]
@rajamatt I am noticing while testing the documentation page that this image seems a bit blurry compared to the other images. Can you update it for a better quality image if possible please
image

File: doc/articles/features/working-with-xaml-hot-reload.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ Mobile targets are currently using a limited version of XAML Hot Reload and do n
 
 Hot Reload is supported by Visual Studio for WinAppSDK and provides support in unpackaged deployment mode.
 
+---
+
 ## Hot Reload Indicator
```
Comment (agneszitte) [inline doc/articles/features/working-with-xaml-hot-reload.md:91]: @rajamatt I am noticing while testing the documentation page that this image seems a bit blurry compared to the other images. Can you update it for a better quality image if possible please
image

**Pull request merged**